fearless/Common/Services/WebSocketService/WebSocketSubscriptionFactory.swift

```diff
@@ -28,41 +30,71 @@ final class WebSocketSubscriptionFactory: WebSocketSubscriptionFactoryProtocol {
                                                                     localStorageIdFactory:
                                                                         localStorageIdFactory)
 
-        let activeEraSubscription = try createActiveEraSubscription(storageKeyFactory: keyFactory,
-                                                                    localStorageIdFactory:
-                                                                        localStorageIdFactory)
-
-        let stakingSubscription = try createStakingSubscription(engine: engine,
-                                                                accountId: accountId,
-                                                                localStorageIdFactory:
-                                                                    localStorageIdFactory)
-
         let bondedSubscription = try createBondedSubscription(accountId: accountId,
                                                               stakingSubscription: stakingSubscription,
                                                               storageKeyFactory: keyFactory)
 
-        let children: [StorageChildSubscribing] = [
-            upgradeV28Subscription,
+        let accountSubscriptions: [StorageChildSubscribing] = [
             accountSubscription,
-            bondedSubscription,
-            activeEraSubscription
+            bondedSubscription
         ]
 
-        let container = StorageSubscriptionContainer(engine: engine,
-                                                     children: children,
-                                                     logger: Logger.shared)
+        let globalSubscriptions: [StorageChildSubscribing] = try createGlobalSubscriptions(
+            keyFactory: keyFactory,
+            localStorageIdFactory: localStorageIdFactory)
+
+        let globalSubscriptionContainer = StorageSubscriptionContainer(engine: engine,
+                                                                       children: globalSubscriptions,
+                                                                       logger: Logger.shared)
+
+        let accountSubscriptionContainer = StorageSubscriptionContainer(engine: engine,
+                                                                        children: accountSubscriptions,
+                                                                        logger: Logger.shared)
 
         let runtimeSubscription = createRuntimeVersionSubscription(engine: engine,
                                                                    networkType: type)
 
-        return [container, stakingSubscription, runtimeSubscription]
+        return [globalSubscriptionContainer,
+                accountSubscriptionContainer,
+                stakingSubscription,
+                runtimeSubscription]
+    }
+
+    private func createGlobalSubscriptions(
```

Comment:
я так понял, в проекте принято
```suggestion
    private func createGlobalSubscriptions(keyFactory: StorageKeyFactoryProtocol,
                                           localStorageIdFactory: ChainStorageIdFactoryProtocol) 
    throws -> [StorageChildSubscribing] {
```

Reply:
Here is a minor problem: if I put parameters the way you suggest, the next line will violate SwiftLint line length rule. I'll clear it out and fix it

Reply:
Will discuss formatting approach at the end of the sprint